src/server/tb/biosecurity-map/size-error/size-error-page-controller.js

```diff
@@ -1,12 +1,11 @@
-import { submitSummaryPage } from '../../check-answers/index.js'
 import { PageController } from '../../../common/controller/page-controller/page-controller.js'
 import { BiosecurityMapAnswer } from '../../../common/model/answer/biosecurity-map/biosecurity-map.js'
 import { TbStateManager } from '~/src/server/tb/state-manager.js'
 import { uploadPlanPage } from '../upload-plan/index.js'
 
 export class SizeErrorPageController extends PageController {
   nextPage() {
-    return submitSummaryPage
+    return { urlPath: '/tb/submit/check-answers' }
```

Comment:
is it correct to just define the urlPath and not pass the page as we normally do? just checking...

Reply:
yes including page creates a cyclic dependency